test/auth/authenticate.test.js

```diff
@@ -0,0 +1,64 @@
+import { getADGroups } from '../../app/auth/authenticate.js'
+import jwt from 'jsonwebtoken'
+import jwksClient from 'jwks-rsa'
+
+jest.mock('jsonwebtoken')
```

Comment:
you are using shadow mock with jest to facilitate mistake of not using dependency injection pattern, this is a frontend hack and not native node approach. Plus I thought we agreed to not use jest and now we are back at jest. There are too many inconsistency here.